packages/frontend/src/hooks/useExplorerRoute.ts

```diff
@@ -98,18 +73,23 @@ export const createMetricPreviewUnsavedChartVersion = (
 
     return {
         ...DEFAULT_EMPTY_EXPLORE_CONFIG,
-        tableName: metric.tableName,
+        tableName: metric.table,
         chartConfig,
         metricQuery: {
             ...DEFAULT_EMPTY_EXPLORE_CONFIG.metricQuery,
-            exploreName: metric.tableName,
+            exploreName: metric.table,
             dimensions: defaultTimeDimension
-                ? [getItemId(defaultTimeDimension)]
+                ? [
+                      getItemId({
+                          name: defaultTimeDimension.field,
+                          table: metric.table,
+                      }),
```

Comment:
```suggestion
                      [
                      getItemId({
                          table: metric.table,
                          name: getFieldIdForDateDimension(
                              defaultTimeDimension.field,
                              defaultTimeDimension.interval,
                          ),
                      }),
                  ]
```

so that the date dimension is selected. the current changes you have will give the field without the time granularity